src/modules/mentors/entities/mentor.entity.ts

```diff
@@ -9,6 +9,10 @@ export class MentorEntity {
   role: string;
   gender: string;
   aboutMe: string;
+  calendlyAccessToken: string;
```

Comment:
Sugiro que essas informações sejam salvos em outra tabela voltada a ter exclusivamente dados do calendly uma Relação ONE-TO-ONE